docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1954,32 +1954,6 @@ The following intervals are supported:
 - years
 - century
 
-#### Example
```

Comment:
./dev/update_function_docs.sh did this?

Reply:
I think it is because you removed the docs (perhaps accidentally) above -- i left a comment